LICENSE.md

```diff
@@ -0,0 +1,194 @@
+Apache License
+==============
+
+_Version 2.0, January 2004_
+_&lt;<http://www.apache.org/licenses/>&gt;_
```

Comment:
So many angl brackets...I know this is a LICENSE, but is this a mistranslation in cut-and-paste?